src/redisearch_api.c

```diff
@@ -414,14 +447,18 @@ static void RS_ResultsIteratorFree(RS_ApiIter* iter) {
   }
   QAST_Destroy(&iter->qast);
   rm_free(iter);
+
+  RWLOCK_RELEASE();
 }
 
 static void RS_ResultsIteratorReset(RS_ApiIter* iter) {
   iter->internal->Rewind(iter->internal->ctx);
 }
 
 static RSIndexOptions* RS_CreateIndexOptions() {
-  return rm_calloc(1, sizeof(RSIndexOptions));
+  RSIndexOptions* ret = rm_calloc(1, sizeof(RSIndexOptions));
+  ret->gcPolicy = GC_POLICY_NONE;
```

Comment:
Why not just make GC_POLICY_NONE the default at 0?

Reply:
Because 0 is default GC